.gitignore

```diff
@@ -0,0 +1,8 @@
+pg_auth_mon.bc
+pg_auth_mon.o
+pg_auth_mon.so
+regression.diffs
+regression.out
+results/
+test
+.gdb_history
```

Comment:
I am sure you want to remove this line.

Reply:
what is wrong with having its here ?

Reply:
well what is the use of this in the final repo, this is only required for your local testing,etc. Or are there some more intentions with it...?